jira_docugen.tex

```diff
@@ -1,67 +1,55 @@
-&
-\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T217}{Full
-Stream Alert Distribution}\tabularnewline
-\protect\hyperlink{lvv-t218---simple-filtering-of-the-lsst-alert-stream}{LVV-T218}
-&
-\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T218}{Simple
-Filtering of the LSST Alert Stream}\tabularnewline
+    \hyperref[lvv-t17]{LVV-T17} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T17}{AG-00-00: Installation of the Alert Generation v16.0 science payload.} \tabularnewline
+    \hyperref[lvv-t18]{LVV-T18} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T18}{AG-00-05: Alert Generation Produces Required Data Products} \tabularnewline
+    \hyperref[lvv-t19]{LVV-T19} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T19}{AG-00-10: Scientific Verification of Processed Visit Images} \tabularnewline
+    \hyperref[lvv-t20]{LVV-T20} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T20}{AG-00-15: Scientific Verification of Difference Images} \tabularnewline
+    \hyperref[lvv-t21]{LVV-T21} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T21}{AG-00-20: Scientific Verification of DIASource Catalog} \tabularnewline
+    \hyperref[lvv-t22]{LVV-T22} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T22}{AG-00-25: Scientific Verification of DIAObject Catalog} \tabularnewline
+    \hyperref[lvv-t216]{LVV-T216} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T216}{Installation of the Alert Distribution payloads.} \tabularnewline
+    \hyperref[lvv-t217]{LVV-T217} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T217}{Full Stream Alert Distribution} \tabularnewline
+    \hyperref[lvv-t218]{LVV-T218} &
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T218}{Simple Filtering of the LSST Alert Stream} \tabularnewline
 \bottomrule
 \end{longtable}
 
-\section{Test Cases}\label{test-cases}
+\newpage
+
+\section{Test Cases}
 
-\subsection{\texorpdfstring{\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T17}{LVV-T17}
-- AG-00-00: Installation of the Alert Generation science
-payload.}{LVV-T17 - AG-00-00: Installation of the Alert Generation science payload.}}\label{lvv-t17---ag-00-00-installation-of-the-alert-generation-science-payload.}
+\subsection{\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testCase/LVV-T17}{LVV-T17}
+    - AG-00-00: Installation of the Alert Generation v16.0 science payload.}\label{lvv-t17}
 
-\begin{longtable}[]{@{}llllll@{}}
+\begin{longtable}[]{llllll}
 \toprule
-Version & Status & Priority & Verification Type & Critical Event &
-Owner\tabularnewline
-\midrule
-\endhead
-1 & Approved & Normal & Test & False & Eric Bellm\tabularnewline
-\bottomrule
+Version & Status & Priority & Verification Type & Critical Event & Owner
```

Comment:
We're told that the “critical event” field is not used by DM — why are we including it here?